### PR TITLE
LPS-85974 Avoid dropdown dissappear too soon

### PR DESCRIPTION
**Pull request by antonio-ortega**
Hi @julien ,

Can you review this PR, please? More information in [LPS-85974](https://issues.liferay.com/browse/LPS-85974).

Don't hesitate to ask me any question you have.

Thanks.

/cc @NemethNorbert, @jbalsas

### CODE REVIEW

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/navigation_interaction.js [hunk 1]
```diff
@@ -44,15 +44,19 @@ AUI.add(
...
 								if (menu) {
 									instance._lastShownMenu = null;
 
 									if (showMenu) {
 										instance._lastShownMenu = menu;
 									}
 
-									menu.toggleClass('hover', showMenu);
-									menu.toggleClass('open', showMenu);
+									instance._debounced = setTimeout(function () {
```
Comment (jbalsas) [inline modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/navigation_interaction.js]: Use `A.debounce` instead

**Comment by antonio-ortega** [comment]
Updated. Thanks for the suggestion.

**Comment by jbalsas** [comment]
Hey @antonio-ortega, @NemethNorbert, this looks better now, but looking at the code, I have a lot of doubts this will work satisfactory.

For one, it likely introduces delays both to show the dropdown and to hide it, which will make it feel very laggy. Additionally, we're not cancelling the toggling, so I'd say that if you go in and out fast (or viceversa), chances are the dropdown will disappear or appear when it shouldn't.

The way to approach this should be:
- Immediately show the dropdown on hover
- On out, set a timer to hide the dropdown, big enough so that you can move comfortably from the trigger to the list, but small enough so that it's not perceptible when really abandoning the dropdown.
- If a new hover is detected, destroy the on-out timer to make sure the dropdown will stay visible

Could you please take a look at this and see if we can improve the solution?

Thanks!

**Comment by jbalsas** [comment]
ci:close

**Pull request closed without merging**